runtime/binding-http/src/test/java/io/aklivity/zilla/runtime/binding/http/internal/streams/rfc7230/client/ConnectionManagementPoolSize1IT.java

```diff
@@ -238,6 +238,7 @@ public void shouldResetRequestAndFreeConnectionWhenRequestWithContentLengthIsRes
         k3po.finish();
     }
 
+    @Ignore("There is no HTTP_MAXIMUM_QUEUED_REQUESTS_NAME")
```

Comment:
If this IT is still relevant, then let's remove `@Ignore` and make it pass, otherwise if not still relevant then we can remove the IT.

Reply:
I don't think it is relevant anymore.